packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -5301,7 +5301,7 @@ curl -v -X POST \
 
 #### Unlock Account with SMS Factor
 
-Starts a new unlock recovery transaction with a user identifier (`username`) and asynchronously sends a SMS OTP (challenge) to the user's mobile phone.  This operation will transition the recovery transaction to the `RECOVERY_CHALLENGE` state and wait for user to [verify the OTP](#verify-sms-recovery-factor).
+Starts a new unlock recovery transaction with a user identifier (`username`) and asynchronously sends a SMS OTP (challenge) to the user's mobile phone.  This operation will transition the recovery transaction to the `RECOVERY_CHALLENGE` state and wait for the user to [verify the OTP](#verify-sms-recovery-factor).
```

Comment:
this actually has some other really minor style issues:

needs to be 'an' instead of 'a' in front of SMS as the abbreviation is most commonly read than spelling it out. 

also  - we try to avoid future tense as much as possible and so as I come across these in the numerous developer docs, I fix them...when they can be fixed:

This operation transitions the recovery.....and waits for the user to....

Reply:
Thanks, that's the kind of advice I need; I wasn't sure how far to go yet.

Reply:
> Thanks, that's the kind of advice I need; I wasn't sure how far to go yet.

Feel free to go ALL THE WAY, Robert! :)